index seperation /split

InjectedPie opened the pull request.

InjectedPie commented:
id locale is also fixed with this pr

rico666 commented:
Can you explain the changes to brs.sitebuild.js - why is all the code for the modals not necessary anymore?

InjectedPie commented:
loadInfoHTML is the same as loadModalHTML I only used it once (now moved to the sidebar context).
The Modaltemplates where unused (Line 52) so I removed them. Basicly its now a modified sitebuild from nxt 1.4.6 (where they had their split) instead of 1.4.18
